packages/@dcl/inspector/src/components/Renderer/Renderer.tsx

```diff
@@ -22,18 +21,19 @@ import { Warnings } from '../Warnings'
 import { CameraSpeed } from './CameraSpeed'
 
 import './Renderer.css'
+import { selectAssetCatalog } from '../../redux/app'
 
 const fixedNumber = (val: number) => Math.round(val * 1e2) / 1e2
 
 const Renderer: React.FC = () => {
   const canvasRef = React.useRef<HTMLCanvasElement>(null)
   useRenderer(() => canvasRef)
   const sdk = useSdk()
-  const dataLayer = useAppSelector(getDataLayer)
-
+  const dispatch = useAppDispatch()
   const [isLoading, setIsLoading] = useState(false)
```

Comment:
This loading state might need to be controlled from the saga, otherwise right now it set to `true` and then to `false` right after the action is dispatched (because the dispatch does not await)

Reply:
will do it in another pr :)